ocp_resources/image_content_source_policy.py

```diff
@@ -7,3 +7,24 @@ class ImageContentSourcePolicy(Resource):
     """
 
     api_group = Resource.ApiGroup.OPERATOR_OPENSHIFT_IO
+
+    def __init_(self, repository_digest_mirrors=None, **kwargs):
+        """
+        Create/Manage ICSP configuration object. API reference:
+        https://docs.openshift.com/container-platform/4.12/rest_api/operator_apis/imagecontentsourcepolicy-operator-openshift-io-v1alpha1.html
+
+        Args:
+            repository_digest_mirrors (list): [{<source>: <mirrors>}, ..., {<source>: <mirrors>}]
+            source (string): the repository that users refer to, e.g. in image pull specifications.
```

Comment:
remove `source` and `mirrors` from the docstring.

Reply:
shouldn't there be a doc for what to pass inside `repository_digest_mirrors` ?